src/features/dashboard/components/AppForm/index.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { ReactNode, useMemo } from 'react';
+
+import { Text } from '@deriv/ui';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import useApiToken from '@site/src/hooks/useApiToken';
+import { appRegisterSchema, IRegisterAppForm } from '../../types';
+import styles from './app-form.module.scss';
+
+type TAppFormProps = {
+  initialValues?: Partial<IRegisterAppForm>;
+  isUpdating?: boolean;
+  renderButtons: () => ReactNode;
+  submit: (data: IRegisterAppForm) => void;
+};
+
+const AppForm = ({ initialValues, submit, renderButtons }: TAppFormProps) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IRegisterAppForm>({
+    mode: 'onBlur',
+    resolver: yupResolver(appRegisterSchema),
+    defaultValues: initialValues,
+  });
+  const { tokens } = useApiToken();
+
+  const adminTokens = useMemo(() => {
+    return tokens.filter((item) => item.scopes.includes('admin'));
+  }, [tokens]);
+
+  return (
+    <React.Fragment>
+      <form role={'form'} className={styles.apps_form} onSubmit={handleSubmit(submit)}>
```

Comment:
Is the form role necessary on an form element? I thought the form element has this by default.